src/error_handler.ts

```diff
@@ -0,0 +1,11 @@
+import { AssertionError } from "assert";
+import chalk from "chalk";
+
+export function errorHandler(err: unknown) {
+	if (err instanceof AssertionError) {
+		console.error(chalk`{red ${err.message}}`);
+	} else if (err instanceof Error) {
```

Comment:
Is it on purpose you removed some of the error handling? This was the old catch block:
```
if (e instanceof assert.AssertionError) {
	console.error(chalk`{red ${e.message}}`);
} else if (e.message.startsWith("Process exited")) {
	const stderr = `${e["stderr"]}`.trim();
	console.error(chalk`{red ${stderr}}`);
} else {
	console.error(chalk`{red ${e.stack}}`);
}
if (e.hint) console.info(e.hint);
process.exit(1);
```